composer.json

```diff
@@ -25,6 +25,9 @@
     "league/csv": "^9.7",
     "ianm/iso-639": "^1.0"
   },
+  "require-dev": {
```

Comment:
This should be an `optional-dependancy`, not a `dev`

Reply:
I don't think there are optional dependencies for Composer: https://github.com/composer/composer/issues/6255

The suggestion seems to be to list it under `suggest` and `conflict`:
```json
    "suggest": {
        "flarum/tags": "This extension works well with flarum/tags."
    },
    "conflict": {
        "flarum/tags": "<2.1.0,>=2.2.0"
    }
```

Reply:
Sorry for the confusion here. `suggest` _could_ be used, but I don't think it's necessary. We certainly don't want to `conflict` with tags, as then it will be uninstallable.

See here for an example of `optional-deps` https://github.com/imorland/follow-users/blob/c7bc7b478826a885096913395e7f08051da0f35c/composer.json#L40

Reply:
Ahh! Gotcha!

Reply:
Shouldn't we keep it under `require-dev` so that we get PHP Intellisense when developing?

Reply:
Nope, Intellisense does not need it :)